### DISCUSSION TITLE
ACP-84: Table Preamble for ACPs

### DISCUSSION THREAD

**Discussion by avalanche-foundation-admin** [avalanche-foundation/ACPs#86] [Discussions]
The current ACP template features a plain-text code block containing "RFC 822 style headers" as `Preamble` (see [What belongs in a successful ACP?](https://github.com/avalanche-foundation/ACPs?tab=readme-ov-file#what-belongs-in-a-successful-acp)). This header includes multiple links to discussions, authors, and other ACPs.

This ACP proposes to replace the `Preamble` code block with a Markdown table format (similar to what is used in [Ethereum EIPs](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md)).

https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/84-table-preamble/README.md

**Comment by patrick-ogrady** [comment]
I thought a bit more about my earlier comment and want to propose that instead of:

| ACP                | 84                                              |
| :----------------- | :---------------------------------------------- |
| **Title**          | Table Preamble for ACPs                         |
| **Author(s)**      | Gauthier Leonard <https://github.com/Nuttymoon> |
| **Discussions-To** | [ACP-84: Table Preable for ACPs](./README.md)   |
| **Status**         | Proposed                                        |
| **Track**          | Meta                                            |

We unify [1] `Status` and `Discussions-To` and [2] replace `<>` with `()` around `Author` info (`<>` don't work well in Markdown):

| ACP                | 84                                              |
| :----------------- | :---------------------------------------------- |
| **Title**          | Table Preamble for ACPs                         |
| **Author(s)**      | Gauthier Leonard (https://github.com/Nuttymoon) |
| **Status**         | Proposed ([Discussion](./README.md))                                 |
| **Track**          | Meta                                            |

A nice byproduct of this approach is that we don't need to show a `Discussion` link for all statuses (might be confusing when an ACP is "Ratified" or "Stale").

**Reply by Nuttymoon** [reply]
I like the idea. `<>` were useless indeed

**Reply by StephenButtolph** [reply]
Would it make sense to have the authors hyper linked? aka:

[Gauthier Leonard](https://github.com/Nuttymoon)

rather than:

Gauthier Leonard (https://github.com/Nuttymoon)

I suppose some people may be better known by their github handles than their actual names.

**Reply by dhrubabasu** [reply]
I'd personally prefer "Gauthier Leonard ([@Nuttymoon](https://github.com/Nuttymoon))" since it matches EIPs and people are known by both their real name and username

**Reply by Nuttymoon** [reply]
I like the idea of following the same format as EIPs here, thanks for the suggestions

**Comment by Nuttymoon** [comment]
Marking this ACP as `Implementable` here: https://github.com/avalanche-foundation/ACPs/pull/100
